app/components/Admin/NaicsCodeSelectionComponent.tsx

```diff
@@ -0,0 +1,46 @@
+import Link from 'next/link';
+import {useRouter} from 'next/router';
+import React from 'react';
+import {ListGroup} from 'react-bootstrap';
+
+interface Props {
+  naicsCodes: {code: string; description: string; id: string}[];
+}
+
+export const NaicsCodeSelectionComponent: React.FunctionComponent<Props> = ({
+  naicsCodes
+}) => {
+  const router = useRouter();
+
+  return (
+    <ListGroup variant="flush">
```

Comment:
If you make the containing rectangle a height-constrained scrollable area, you'll want to ensure to set `tab-index="0"` so it can be interacted with using keyboard focus:

```
<div className="scrollable" tabIndex={0}>
```

I don't think you can target the scrollbar pseudoselectors using inline styles (for ex. on the Bootstrap ListGroup), so you may need to "line" it with a container div and make that the height-constrained scrollable area as I've shown.